docs/style.md

```diff
@@ -106,6 +106,34 @@ use crate::types::KnownPeerState;
 can't split imports into groups automatically, and doing that manually
 consistently is a chore.
 
+### Derives
+
+When deriving an implementation of a trait, specify a full path to the traits provided by the
+external libraries:
+
+```rust
+// GOOD
+#[derive(Copy, Clone, serde::Serialize, thiserror::Error, strum::Display)]
```

Comment:
Do you think we should be using `BorshDerive` or `borsh::BorshDerive`? (Especially if we make an exception for `serde`?)

Reply:
Could we add `pub use BorshSerialize as Serialize;` (or whatever) to borsh so that we can write `borsh::Serialize`?

Reply:
It would be awesome if we could do that. Can we? This doesn't even need to be a breaking change – we could re-export for now and do a proper rename for the next breaking change if ever.